Makefile

```diff
@@ -656,13 +714,13 @@ tables/all_production: data/all-production/product
 	rm $$tmp
 	@$(call load-sql,data/all-production/rollup.sql)
 
-tables/company_revenue: data/company/years
+tables/company_revenue: data/company-revenue/output
 	@$(call drop-table,company_revenue)
 	tmp=$^/all.ndjson; \
 	for company_filename in $^/????.tsv; do \
 		filename="$${company_filename##*/}"; \
 		COMPANY_YEAR="$${filename%%.*}"; \
-		$(tito) -r tsv --map ./data/company/transform.js \
+		$(tito) -r tsv --map ./data/company-revenue/transform.js \
```

Comment:
Line 717 and 723 already exist on master

[proof](https://github.com/18F/doi-extractives-data/blob/dev/Makefile#L717-L7230)